### DISCUSSION TITLE
How to let output terminal NOT show message and warnings of gnuplot ?

### DISCUSSION THREAD

**Discussion by Linhuihang** [alandefreitas/matplotplusplus#277] [Q&A]
Gnuplot prints its message and warnings to output terminal. How to disable?

gnuplot> se
           ^
         line 0: unrecognized option - see 'help set'.

gnuplot> t terminal wxt title "Figure 1" size 560,420 enhanced font "Sans,10"
         ^
         line 0: invalid command

gnuplot> se
           ^
         line 0: unrecognized option - see 'help set'.

gnuplot> t terminal wxt title "Figure 1" size 560,420 enhanced font "Sans,10"
         ^
         line 0: invalid command

Warning: empty x range [0:0], adjusting to [-1:1]
Warning: empty y range [2.23126e+06:2.23126e+06], adjusting to [2.20895e+06:2.25357e+06]

**Comment by alandefreitas** [comment]
The pipe is created here

https://github.com/alandefreitas/matplotplusplus/blob/8dbea7d359f7b4f456bca7a6015c32b61ad728f4/source/matplot/backend/gnuplot.cpp#L66-L71

But there's no option to redirect the errors somewhere else. 

You could patch this backend to do that or patch the plots so that there are no errors.
